spec/system/school/courses/target_version_management_spec.rb

```diff
@@ -13,250 +11,109 @@
-        target_1.content_versions.reload
-        expect(target_1.latest_content_versions.count).to eq(4)
-        expect(ContentBlock.find_by(id: cb_id_to_delete)).to eq(nil)
-        expect(target_content_versions.count).to eq(11)
+      expect(page).to have_text('#2')
+      target_1.current_target_version.content_blocks.last.delete
+      target_1.current_target_version.touch
 
-        # Delete an old content block
-        cb_id_to_delete = target_1.latest_content_versions.where(sort_index: 2).first.content_block_id
-        accept_confirm do
-          within("div[aria-label='markdown editor for #{cb_id_to_delete}']") do
-            find_button('Delete block').click
-          end
-        end
-        expect(page).to have_selector('.content-block__content', count: 3)
-        target_1.content_versions.reload
-        expect(target_1.latest_content_versions.count).to eq(3)
-        # the content block should not be destroyed
-        expect(ContentBlock.find_by(id: cb_id_to_delete)).to_not eq(nil)
-        expect(target_1.content_versions.count).to eq(10)
+      click_button "Select version #{target_1.current_target_version.id}"
+      click_button "Select version #{target_version_v1.id}"
 
-        # Change content block sorting
-        block_to_move = target_1.latest_content_versions.where(sort_index: 2).first.content_block_id
-        within("div[aria-label='file editor for #{block_to_move}']") do
-          find_button('Move up').click
-        end
+      click_button 'Restore this version'
+      expect(page).to have_text('A new version has been created')
+      dismiss_notification
 
-        target_1.content_versions.reload
-        expect(target_1.latest_content_versions.count).to eq(3)
-        expect { target_1.latest_content_versions.where(sort_index: 3).first.content_block.block_type }.to eventually(eq 'embed')
-        expect { target_1.latest_content_versions.where(sort_index: 1).first.content_block.block_type }.to eventually(eq 'file')
-      end
+      expect(page).to have_text('#3')
+      expect(target_1.current_target_version.content_blocks.order(:sort_index).map { |cb| cb.slice(:block_type, :content, :sort_index) }).to match_array(content_blocks_v1)
     end
+  end
 
-    scenario 'restores to an old version' do
-      sign_in_user school_admin.user, referer: curriculum_school_course_path(course)
-
-      # View an old version of the target
-      find('.target-group__target', text: target_1.title).click
-      expect(page).to have_button('Edit')
-      expect(page).to_not have_selector('.add-content-block--open', count: 1)
-
-      click_button format_date(Date.today)
-
-      # There should be two options in the dropdown.
-      expect(page).to have_selector('.target-editor__version-dropdown-list-item', count: 2)
-
-      within('#version-selection-list') do
-        find('li', text: format_date(3.days.ago)).click
-      end
+  context 'course author visits target version page' do
```

Comment:
This `context` is also devoid of any set-up. The scenario can simply be `course author saves the current version of a target`.